config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/BundleBuilder.java

```diff
@@ -33,18 +35,25 @@
     private final BundleEntityBuilder bundleEntityBuilder;
     private final DependencyBundleLoader dependencyBundleLoader;
     private final DocumentTools documentTools;
+    // FIXME remove when set keystore to be built in a proper place
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this comment. ([squid:S1134](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1134))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=76&issues=AWZZr2uf9tx2us2yAaRk&open=AWZZr2uf9tx2us2yAaRk)

[ISSUE_KEY]: AWZZr2uf9tx2us2yAaRk